src/main/resources/META-INF/plugin.xml

```diff
@@ -1,6 +1,6 @@
 <idea-plugin>
     <id>io.coala.jetbrains</id>
-    <name>coala Code Analysis</name>
+    <name>coala</name>
```

Comment:
I think we need a better name, since this plugin is not official yet, taking up the base `coala` name is not good. Maybe somthing that indicates that this enables `coala` in editor, something like `coala brigde` etc.

Reply:
Maybe then leave it as `coala Code Analysis`? We can rename it later as-well.

Reply:
Sounds good to me.